test/test_preprocessing.py

```diff
@@ -1,10 +1,11 @@
-from preprocessing.preprocessing import get_preprocessed_brfss_train_test_split
+from preprocessing.preprocessing import get_train_test_split, get_preprocessed_brfss_dataset
 
 
 class TestPreprocessing:
 
     def test_get_preprocessed_train_test_split(self):
-        train_data, test_data, train_target, test_target = get_preprocessed_brfss_train_test_split()
+        dataset, target = get_preprocessed_brfss_dataset()
+        train_data, test_data, train_target, test_target = get_train_test_split(dataset, target)
```

Comment:
If you previously used the get_preprocessed_brfss_train_test_split() method, you must now first call the preprocessing method you want to use (i.e. which encoding and sampling variant) and then call the preprocessing.get_train_test_split() function.